setup.cfg

```diff
@@ -2,7 +2,18 @@
 ignore =
     conf.py
 
-
-
 [bdist_wheel]
 universal = 0
+
+[flake8]
+ignore =
+    # black takes care of line length
+    E501,
+    # black takes care of where to break lines
+    W503,
+    # black takes care of blank lines
+    D202,
```

Comment:
I have almost the same ignores in a small script that I run locally: `flake8 --ignore=E203,E231,E501,W503`. I have these, because there can be conflicts, where black insists on a syntax that flake8 complains about.

The only difference is that I do not ignore `D202` for the blank lines. I have not seen conflicts between black and flake8 here, so it was not needed for me yet. But I don't mind having it in here.

Reply:
I added this on a repository back in 2018, which maybe `black` was not that stable, I will remove it 👍🏾